settings.yml

```diff
@@ -2,7 +2,7 @@ category_icon_list:
   default: "help,question-circle,#CC0000,partial|"
   type: "list"
   list_type: "simple"
-  description: 'Enter comma-delimited configuration for categories, in the format "slug,icon,color,match". Colour in format #123456 or "categoryColor" to use the default color for the category (same as the Badge color). If  match is "partial" then the slug need only partially match the category-slug, otherwise an exact match is required'
+  description: '  Enter comma-delimited configuration for categories<br><br>Format: "slug,icon,color,partial"<br><br>slug — The category slug.<br><br>icon — Can be either (1) a Font Awesome icon, (2) a singular unicode character "😀", or (3) a forum emoji formatted like ":grinning:"<br><br>color (optional) — Color for the icon. Use a hex code with the "#" character. Default color is the category color.<br><br>partial (optional) — A color must be set first to activate this. Write "partial" if the slug only needs to partially match the category-slug, otherwise an exact match is required. Useful for subcategories with identical names.'
```

Comment:
> Default color is the category color.

I don't think this is a bad idea per se, but it is a breaking change. For existing users of the component, it will result in a change in behaviour. Except for sites that were previosuly using `categoryColor`.

Is there a strong reason to make this change? If not, I think we should keep the functionality as is.